src/js/checklist.js

```diff
@@ -58,10 +58,45 @@ function populateChecklistFromLocalStorage(checkboxSelector) {
 
 function processChecklist() {
 	var checkboxSelector = '.c-checklist__checkbox input[type="checkbox"]';
-
 	populateChecklistFromLocalStorage(checkboxSelector);
 	processChecklistClick(checkboxSelector);
 }
 
 openLinkedCheckListItem();
 processChecklist();
+
+function registerToggleButton(buttonEl, parentEl) {
+	if (buttonEl && parentEl) {
+		var details = parentEl.querySelectorAll("details");
+		buttonEl.addEventListener("click", function handleToggleButtonClick(event) {
+			var target = event.target;
+			if (target.hasAttribute("data-open") === true) {
+				target.removeAttribute("data-open");
```

Comment:
Can we instead use `aria-expanded="true"` in place of data-open? I think that will help convey meaningful information to the assistive technology about the state of the button as well, and helps with the logic as well.
For reference: https://adrianroselli.com/2020/05/disclosure-widgets.html#Script